src/lib/markbind/src/utils/nunjuckUtils.js

```diff
@@ -7,8 +13,14 @@ function preEscapeRawTags(pageData) {
 }
 
 module.exports = {
-  renderEscaped(nunjucks, pageData, variableMap = {}, options = {}) {
+  renderEscaped(pageData, variableMap = {}, options = {}, env = commonEnv) {
     const escapedPage = preEscapeRawTags(pageData);
-    return nunjucks.renderString(escapedPage, variableMap, options);
+    return env.renderString(escapedPage, variableMap, options);
+  },
+  render(pageData, variableMap = {}, options = {}, env = commonEnv) {
```

Comment:
how about having a separate `renderUnescaped`? Would avoid having to export the `env` configurations which could be kept in this file.

Reply:
or you could change the fourth option `env` to `autoEscape = false` and do the rendering accordingly

In any case I think its not such a good idea to have a utility file requiring configuration with another one of its exports ( `unescapedEnv` )

Reply:
I was kind of on the fence with this since there are two kinds of escaping (`{%raw%}` vs HTML characters) in here, I also wanted to keep the original signature of accepting env as an argument. I suppose it would be clearer to simply make a separate method that uses the unescapedEnv, following the first suggestion. But we would have to name it something other than `renderUnescaped` since it implies that it is the opposite of `renderEscaped` which it isn't.

Reply:
> To summarize:

Sounds good 👍 I would personally leave `renderUnescaped` as an option in `renderRaw` so that its clearer that we are adding the extra `{% raw/endraw %}`'s in the call though. Since its actually more of `renderRaw+NoAutoEscape`.

Reply:
Good suggestion, it's clearer and less work